config/webhook-setup/proxy/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+namespace: multi-cluster-traffic-controller-system
+
+resources:
+- namespace.yaml
+- deployment.yaml
+- service.yaml
+- ../control/ingress.yaml
+
+configMapGenerator:
+  - name: webhook-proxy-conf
+    files:
+      - nginx.conf
+
+secretGenerator:
+- name: ingress-tls
```

Comment:
Looks like this will have to be called `mctc-ingress-tls`. The default kustomize config uses a name prefix of `mctc`, but this one isn;t, and isn;t called from another one, so you need to be explicit with the name here if you are sharing the same ingress setup for both.